make.inc.macosx_gcc-10

```diff
@@ -0,0 +1,54 @@
+# Makefile variable overrides for Mac OSX compilation with GCC v.10.*
+#
+# Use this if you'll need to link against gfortran.
+#
+# Copy this file to make.inc, and if needed edit for your setup.
+#
+# Don't forget also to adjust g++ to g++-10 at start of setup.py
```

Comment:
this is a dead (obsolete) comment. I just removed equiv from gcc-8; could you do the same?

Reply:
I will remove this.